### PR TITLE
[TASK] Move the PHPMD check from Travis to GitHub actions

### PR DESCRIPTION
**Pull request by oliverklee**
Part of #818.

### CODE REVIEW

**Comment by oliverklee** [comment]
I've used YAML references and anchors. Looks like I've broken the build … I'll look into it.

**Comment by oliverklee** [comment]
Oh:

> Your workflow file was invalid: The pipeline is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'php-73'

**Comment by oliverklee** [comment]
I'll keep looking for other possible solutions.

**Comment by oliverklee** [comment]
Note to myself: Maybe I can make an action of this. https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idsteps

**Comment by oliverklee** [comment]
Okay, so after a fair bit of research, these are my findings:
1. [There is no support for nested workflows.](https://github.community/t5/GitHub-Actions/Nested-workflows-in-GitHub-actions/td-p/39284)
1. [There is no support for anchors.](https://github.community/t5/GitHub-Actions/Support-for-YAML-anchors/td-p/30336)
1. [It is possible to (mis-)use the matrix feature for some things, e.g., for the static analysis steps that use composer caching and that use the same PHP version.](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix)

I'll look into going route number 3.

**Comment by oliverklee** [comment]
This is ready for review now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#776](https://github.com/MyIntervals/emogrifier/pull/776))
 
 ### Changed
-- Move the code sniffing from Travis CI to GitHub actions
-  ([#832](https://github.com/MyIntervals/emogrifier/pull/832))
+- Move the code quality checks from Travis CI to GitHub actions
+  ([#832](https://github.com/MyIntervals/emogrifier/pull/832),
+  [#833](https://github.com/MyIntervals/emogrifier/pull/833))
```
Comment (JakeQZ) [inline CHANGELOG.md]: This should be 834.

Comment (oliverklee) [reply CHANGELOG.md]: Good catch! I've pushed the correspondinge change.

**Pull request merged**